python/sglang/srt/function_call/pythonic_detector.py

```diff
@@ -125,6 +136,9 @@ def parse_streaming_increment(
         Buffers input until a complete pythonic tool call (from [ to ]) is found,
         then parses and emits any detected calls.
         """
+        # remove unexpected <|python_start|> and <|python_end|> for llama4
+        new_text = self._text_strip(new_text)
```

Comment:
Is there a pattern for when <|python_start|> or <|python_end|> will happen? Will it always be a single token?

`new_text` will always be one token. It will fail to match if the new_text is like `<|pyhton_`, then the next one is `start|>`